spicy/toolchain/src/compiler/codegen/parser-builder.cc

```diff
@@ -1241,7 +1242,7 @@ struct ProductionVisitor : public production::Visitor {
                 e = synchronize_after->expression();
 
             if ( e ) {
-                const auto id = "synchronize";
+                const auto id = hilti::util::fmt("synchronize_%u", std::hash<Meta>{}(e->meta()));
```

Comment:
Does this actually need to be considering the location/meta, or does it just need to be unique? If the latter, better to use `uniquer()->get("synchronize")`, for consistency and to avoid depending on the content of meta.

Reply:
It doesn't and the uniquer seems to see enough to handle this correctly, fixed.